guides/web-apps/html-view-guidelines/README.md

```diff
@@ -98,15 +98,21 @@ Here, it would not make sense to break it up into the respective parts because t
 
 ### Guideline 3: Attaching the Content to Slots
 
-To give clearer semantic meaning to the content of one application when mixing with other applications, explicit slot names are used. When not using explicit slot names, the elements at the root will get implicit slot names and look like this: `<slot name="MyApp/0"></slot>`, the zero is simply the index of the element in the view. With an explicit slot name, it becomes much clearer what kind of element it is: `<slot name="MyApp/MainHeadline"></slot>`.
+To have better flexibility in blending - to be able to distribute an element individually - the element needs to be assigned to a [named slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot#named-slot), like `<button slot="myapp/submitbutton">Submit</button>`. Such an element could be distributed via `<slot name="myapp/submitbutton"></slot>` in the Shadow DOM.
+Please prefix a slot name with the app name, to avoid collisions.
+
+When no slot name is provided for an element it will be distributed in ["default slot"](https://w3c.github.io/webcomponents/spec/shadow/#shadow-tree-slots): `<slot></slot>`.
+Please note, that text nodes will also be distributed there.
 
-Slot names are added as attributes to the elements like so: `<button slot="MyApp/SubmitButton">Submit</button>`.
 
 ### Guideline 4: Create the Layout in `declarative-shadow-dom`
 
-As outlined in guideline 1, the layout of the HTML view should be included within the `<template is="declarative-shadow-dom">`. There is one exception to this, if the default layout of the content equals the targeted look, both slot attributes and `declarative-shadow-dom` can be omitted.
+As outlined in guideline 1, the layout of the HTML view should be included within the `<template is="declarative-shadow-dom">`.
+
+There is one exception to this. Both slot attributes and `declarative-shadow-dom` can be omitted if the view contains only non-visual elements, or all elements should be distributed bulked together in default slot. The latter is rarely the case but may be useful for example for prototyping.
+
```

Comment:
can be omitted if the view contains only non-visual elements, or all elements should be distributed bulked together in default slot 

->

can be omitted if the view only contains non-visual elements or if all elements should be bulked together in the default slot